src/kOS/Module/kOSProcessor.cs

```diff
@@ -75,6 +75,19 @@ public string Tag
         [KSPField(isPersistant = true, guiActive = false)]
         public int MaxPartId = 100;
```

Comment:
I'd say get rid of it.  A search of the code shows that it's literally not used anywhere.  This is the only place it gets mentioned.  I have no clue why it's there.

It is, however, mentioned in the tutorial craft file, here: doc/source/_images/tutorials/quickstart/MyFirstRocket.craft , but that's only because being a KSPField, KSP automatically spits it out into the craft file, and when that tutorial was written it just took the craft file from KSP.  (So it would have to be removed from there too).

Which brings to mind a question.  If a save file or craft file contains a KSPField that got removed between versions, does it break the ability to reload the save?  If so then we may have a problem with removing this.